sys-process/bashtop/bashtop-0.8.29.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Resource monitor that shows usage and stats"
+HOMEPAGE="https://github.com/aristocratos/bashtop"
+SRC_URI="https://github.com/aristocratos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="curl lm-sensors sysstat"
+DOCS=( README.md )
+
+DEPEND="${RDEPEND}"
+RDEPEND="curl? ( >=net-misc/curl-7.16 )
```

Comment:
Conditional runtime dependencies are [controversial](https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001) but I guess for something written in bash, it doesn't matter that much.

You may still want to use optfeature from the [eutils](https://devmanual.gentoo.org/eclass-reference/eutils.eclass/) eclass in pkg_postinst() though.

Reply:
Okay, I will go through it and make changes accordingly.